examples/sst2/configs/default.py

```diff
@@ -42,6 +42,5 @@ def get_config():
   config.num_epochs = 10
 
   config.seed = 0
-  config.deterministic = False
```

Comment:
How is this change connected to using Optax?

Reply:
Ah, got it.

Personally, I prefer being always explicit about that flag, so the change totally LGTM.